net/BUILD.gn

```diff
@@ -2545,17 +2545,17 @@ test("net_unittests") {
     "base/prioritized_dispatcher_unittest.cc",
     "base/prioritized_task_runner_unittest.cc",
     "base/priority_queue_unittest.cc",
-    "base/proxy_server_unittest.cc",
-    "base/proxy_string_util_unittest.cc",
-    "base/registry_controlled_domains/registry_controlled_domain_unittest.cc",
-    "base/scheme_host_port_matcher_rule_unittest.cc",
+    # "base/proxy_server_unittest.cc",
```

Comment:
Use the same pattern of removing sources in a separate conditional block that subtracts from the sources instead of commenting them out

Reply:
Well, this is an initial audit of all those tests. The purpose of commenting all failed test out is to ensure when running "./net_unittests", all existing tests pass. This ensures that any subsequent test failures are directly caused by changes we introduce, rather than existing problems. For example, when Esmael updates //build config, we can clearly see which tests fail specifically due to those updates.

Additionally, when we fix the testS, we can either uncomment them, or move them to the conditional block like what we did last time to disable them. In shorts, this is just to mark them as a "to be fixed".

Reply:
We can do the same thing with a conditional block with the added benefits of having an explicit list of tests that need to be looked at all in one place and no inline modifications to upstream code.

Reply:
I'm not sure this change optimizes our workflow. We frequently need context from tests before and after a failure, which are often related. Grouping all tests together may make it harder to find what we need. "no inline modifications to upstream code" is the eventual goal after we fix/disable them using the conditional block. Doing that right now requires some tedious work and I don't think it worths.

Reply:
I'm not sure if this was unclear but I'm saying to create groups of disabled tests per GN file, not across all of them. You lose a little bit of context from the conditionals but otherwise nothing is lost

Not doing this makes it hard to distinguish what is disabled upstream vs what we explicitly disabled—sometimes upstream build code will just comment out a source but leave it in the list of sources like this. If a future person comes in and wants to fix a test, we don't want them accidentally wasting time trying to fix a test that's also broken upstream.

I won't harp on this because I don't want to block progress on this but considering it's low effort and (I think) helpful, I think it's worth doing. If you still disagree you can leave it as is.